src/System.Net.Http/tests/StressTests/HttpStress/StressClient.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Security;
+using System.Net.Sockets;
+using System.Threading;
+using System.Threading.Tasks;
+using RequestContext = HttpStress.ClientOperations.RequestContext;
+
+namespace HttpStress
+{
+    public class StressClient : IDisposable
+    {
+        private readonly CancellationTokenSource _cts;
+        private readonly Task _clientTask;
+
+        // TOCONSIDER: configuration class to avoid threading so many parameters
```

Comment:
nit ``` // TO CONSIDER:```

Reply:
It's kind of standard, used in same way as TODO in code comments